views/providers.tpl

```diff
@@ -106,6 +106,37 @@
                             </div>
                         </div>
 
+                        <div class="middle aligned row">
+                            <div class="right aligned four wide column">
+                                <label>BetaSeries</label>
+                            </div>
+                            <div class="one wide column">
+                                <div id="betaseries" class="ui toggle checkbox provider">
+                                    <input type="checkbox">
+                                    <label></label>
+                                </div>
+                            </div>
+                            <div class="collapsed column">
+                                <div class="collapsed center aligned column">
+                                    <div class="ui basic icon" data-tooltip="French Trakt equivalent that provides TV Shows only subtitles." data-inverted="">
+                                        <i class="help circle large icon"></i>
```

Comment:
I would like to have a warning sign also with a tooltip saying what should be expected from this provider in term of release group match and probably to get out of sync subtitles. Would it be possible to add this? Thanks again!

Reply:
What do you think of _`This provider only uses the subtitle filename to find release group matches, subtitles may thus be out of sync.`_ in a warning tooltip?

I also need to add the available languages in the help tooltip, is this _`FR/EN provider for TV Shows only.`_ better than the actual message? Because I don't think that "being a french Trakt equivalent" is relevant in the choice of providers.

Reply:
That's perfect in my opinion. Both suggestion meet the objective of giving the user enough information to make a relevant choice.